### PR TITLE
awsweeper 0.2.0 (new formula)

### PR DESCRIPTION
**Pull request by nysthee**
AWSweeper is a tool to clean out your AWS account

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/awsweeper.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+class Awsweeper < Formula
+  desc "Cleat out your AWS account with this convenient tool"
```
Comment (igas) [inline Formula/awsweeper.rb]: typo

Comment (nysthee) [reply Formula/awsweeper.rb]: fixed thanks for spotting this

**Comment by javian** [comment]
@brewtest test this please

**Comment by javian** [comment]
@BrewTestBot test this please

File: Formula/awsweeper.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    awsweeper_path.install buildpath.children
+
+    cd awsweeper_path do
+      system "go", "build"
+      bin.install "awsweeper"
+    end
+  end
+
+  test do
+    assert_match "0.1.1", shell_output("#{bin}/awsweeper --version ")
```
Comment (fxcoudert) [inline Formula/awsweeper.rb]: We need a test that exercises the some of the functionality of the app.

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/awsweeper.rb (hunk 2 above)
Comment (nysthee) [reply Formula/awsweeper.rb]: Hi @fxcoudert thanks for the feedback. The tool is for cleaning up resources on an AWS account so setting something like that up in a test is quite cumbersome. Can you advise on how to mock that? Or point me to a good example of what is expected?

Comment (igas) [reply Formula/awsweeper.rb]: Maybe we can check the output for invalid/not provided credentials?

Comment (nysthee) [reply Formula/awsweeper.rb]: I added the proposed test.

**Comment by nysthee** [comment]
hi @fxcoudert  I added a test.

**Pull request closed without merging**